docs/howto/operator/gcp/automl.rst

```diff
@@ -0,0 +1,172 @@
+    :end-before: [END howto_operator_automl_create_model]
+
+To get existing model one can use
+:class:`~airflow.contrib.operators.gcp_automl_operator.AutoMLGetModelOperator`.
+
+.. exampleinclude:: ../../../../airflow/contrib/example_dags/example_gcp_automl_operators.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_get_model]
+    :end-before: [END howto_operator_get_model]
+
+Once a model is created it could be deployed using
+:class:`~airflow.contrib.operators.gcp_automl_operator.AutoMLDeployModelOperator`.
+
+.. exampleinclude:: ../../../../airflow/contrib/example_dags/example_gcp_automl_operators.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_deploy_model]
+    :end-before: [END howto_operator_deploy_model]
+
+If you wish to delete a model you can use
+:class:`~airflow.contrib.operators.gcp_automl_operator.AutoMLDeleteModelOperator`.
+
+.. exampleinclude:: ../../../../airflow/contrib/example_dags/example_gcp_automl_operators.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_automl_delete_model]
+    :end-before: [END howto_operator_automl_delete_model]
+
+Making Predictions
+^^^^^^^^^^^^^^^^^^
+
+To obtain predictions from Google Cloud AutoML model you can use
+:class:`~airflow.contrib.operators.gcp_automl_operator.AutoMLPredictOperator` or
+:class:`~airflow.contrib.operators.gcp_automl_operator.AutoMLBatchPredictOperator`. In the first case
+the model must be deployed.
+
+.. exampleinclude:: ../../../../airflow/contrib/example_dags/example_gcp_automl_operators.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_prediction]
+    :end-before: [END howto_operator_prediction]
+
+.. exampleinclude:: ../../../../airflow/contrib/example_dags/example_gcp_automl_operators.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_batch_prediction]
+    :end-before: [END howto_operator_batch_prediction]
+
+Listing And Deleting Datasets
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can get a list of AutoML models using
+:class:`~airflow.contrib.operators.gcp_automl_operator.AutoMLDatasetListOperator`. The operator returns list
+of datasets ids in :ref:`XCom <concepts:xcom>` under ``dataset_id_list`` key.
+
+.. exampleinclude:: ../../../../airflow/contrib/example_dags/example_gcp_automl_operators.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_list_dataset]
+    :end-before: [END howto_operator_list_dataset]
+
+To delete a model you can use :class:`~airflow.contrib.operators.gcp_automl_operator.AutoMLDatasetDeleteOperator`.
+The delete operator allows also to pass list or coma separated string of datasets ids to be deleted.
+
+.. exampleinclude:: ../../../../airflow/contrib/example_dags/example_gcp_automl_operators.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_operator_delete_dataset]
+    :end-before: [END howto_operator_delete_dataset]
```

Comment:
A section with references to external documentation (Product documentation API Client) is missing.

Reply:
Should we add it to Munchkin or is it already present?